pom.xml

```diff
@@ -72,7 +72,7 @@
       <dependency>
         <groupId>com.fasterxml.jackson.core</groupId>
         <artifactId>jackson-databind</artifactId>
-        <version>2.9.10</version>
+        <version>2.9.10.1</version>
       </dependency>
```

Comment:
why is not this declaration reused from kie-parent?

Reply:
@yurloc we should remove this from the pom.xml and keep kie-parent to manage it

Reply:
Yes, @cuijulian is working on that.